src/Ocis.php

```diff
@@ -786,4 +788,26 @@ public function createGroup(string $groupName, string $description = "")
         }
     }
 
+    /**
+     * delete an existing group (if the user has the permission to do so)
+     *
+     * @param Group $group
+     * @return void
+     * @throws BadRequestException
+     * @throws ForbiddenException
+     * @throws NotFoundException
+     * @throws UnauthorizedException
+     * @throws \InvalidArgumentException
+     * @throws HttpException
+     */
+    public function deleteGroup(Group $group): void
+    {
+        $apiInstance = new GroupApi($this->guzzle, $this->graphApiConfig);
+        try {
+            $apiInstance->deleteGroup($group->getId());
+        } catch (ApiException $e) {
+            throw ExceptionHelper::getHttpErrorException($e);
+        }
```

Comment:
any reason for not handling ODataError?

Reply:
delete group returns void as it's return type or it throws an exception. It doesn't throw any object and thus can't be checked whether the return value is of `OdataError instance`

Reply:
i can see ODataError inside libregraph deletegroup function. Please be sure what is actual behavior

Reply:
I can see that. But `Odata error ` isn't caught as an Exception in our code. As far as I know, once the API Exception is caught, it can't test whether the variable is of  `OdataError` instances as the function return value is void only (though the exception inside libregraph returns OdataError).

Reply:
@S-Panta is right, `deleteGroup` does not return anything like other functions, so there is no odata error